pom.xml

```diff
@@ -77,7 +77,7 @@
         <native-maven-plugin.version>0.9.23</native-maven-plugin.version>
         <micronaut.aot.version>2.0.0-M3</micronaut.aot.version>
         <micronaut.test.resources.version>2.0.0-M10</micronaut.test.resources.version>
-        <micronaut.openapi.version>5.0.0-M6</micronaut.openapi.version>
+        <micronaut.openapi.version>5.0.0-SNAPSHOT</micronaut.openapi.version>
```

Comment:
Should this be M9?

Reply:
I think we need a new release, it requires this PR: https://github.com/micronaut-projects/micronaut-openapi/pull/1089

Reply:
I will release a new OpenAPI milestone and update this PR.